CHANGELOG.md

```diff
@@ -1,10 +1,46 @@
+## 0.50.0
```

Comment:
In the past we've put the beta designation in the Changelog and removed it as part of the final release process, but I don't have a strong opinion about whether we should keep doing that or not.